Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "flurry"
 version = "0.3.1"
```

Comment:
This is definitely a breaking change, but I'm okay with that. Just leaving this as a reminder to myself.